rpcserver.go

```diff
@@ -5703,11 +5703,16 @@ func (r *rpcServer) ListPayments(ctx context.Context,
 
 // DeleteAllPayments deletes all outgoing payments from DB.
 func (r *rpcServer) DeleteAllPayments(ctx context.Context,
-	_ *lnrpc.DeleteAllPaymentsRequest) (*lnrpc.DeleteAllPaymentsResponse, error) {
+	req *lnrpc.DeleteAllPaymentsRequest) (
```

Comment:
nit: consider squashing with previous commit (proto changes). (similarly for failed htlcs)

Reply:
Squashed proto changes